### PR TITLE
Use docker exec instead of docker cp to create run script for Kafka native DevService

### PR DESCRIPTION
**Pull request by xstefank**
This also requires https://github.com/ozangunalp/kafka-native/pull/144 to be pushed to quay.io. cc/ @ozangunalp 

With this change I was able to successfully run Kafka native DevService in DevSpaces/Che!

config
```
quarkus.kafka.devservices.provider=kafka-native
```

This would really help us to get the Quarkus super-heroes workshop to be doable in DevSpaces (fully online).

cc/ also @danieloh30 since I heard that this was a blocker for you.

### CODE REVIEW

**Comment by ozangunalp** [comment]
@xstefank that string format is hard to follow, but it looks good!

Can we do the same for the redpanda container?

**Comment by ozangunalp** [comment]
@xstefank also, the change itself doesn't require the kafka-native release right? We can cut it after? 

Also you can test with the kafka-native `latest-snapshot` tag that is built & pushed from the main.

**Comment by xstefank** [comment]
@ozangunalp redpanda has itself similar issue as we fixed in your kafka-native container (permissions for random UID) so until they fix it in their container, we can't do anything on our side.

Add release, yes, this is just a different way to create run.sh script in container so it shouldn't matter. But I see test failures so probably it does?

**Comment by ozangunalp** [comment]
`KafkaDevServicesDevModeTestCase.sseStream` is a failing test, not even flaky apparently. I am looking at it. So it is irrelevant for this change.

File: extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/KafkaNativeContainer.java [hunk 1]
```diff
@@ -59,10 +57,13 @@ protected void containerIsStarting(InspectContainerResponse containerInfo, boole
...
         }
 
-        //noinspection OctalInteger
-        copyFileToContainer(
-                Transferable.of(cmd.getBytes(StandardCharsets.UTF_8), 0777),
-                STARTER_SCRIPT);
+        // docker exec since docker cp doesn't work with kubedock yet
+        try {
+            execInContainer("sh", "-c",
+                    String.format("echo -e \"%s\" >> %s && chmod 777 %s", cmd, STARTER_SCRIPT, STARTER_SCRIPT));
```
Comment (gsmet) [inline extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/KafkaNativeContainer.java:63]: The following might be a little less confusing (untested):
```suggestion
                    String.format("echo -e \"%1$s\" >> %2$s && chmod 777 %2$s", cmd, STARTER_SCRIPT));
```

Comment (xstefank) [reply extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/KafkaNativeContainer.java:63]: Thanks

**Review by ozangunalp — APPROVED** [review]
